_docs/06-best-practice-pull-request.md

```diff
@@ -4,23 +4,45 @@ permalink: /docs/pull-request/
 modified: 2016-12-24T15:01:43-04:00
 ---
 
-There are several ways to setup Git for committers and contributors. Contributors can safely setup Git any way they choose but committers should take extra care since they can push new commits to the master at Apache and various policies there make backing out mistakes problematic. Therefore all but very small changes should go through a PR, even for committers. To keep the commit history clean take note of the use of --squash below when merging into apache/master.
+There are several ways to setup Git for committers and contributors.<br />
+Contributors can safely setup Git any way they choose but committers should take extra care since they can push new commits to the master at Apache and various policies there make backing out mistakes problematic. Therefore all but very small changes should go through a PR, even for committers.<br />
```

Comment:
IMO, we should drop this paragraph. To complement, we'd better move the last warning to the committer repository set-up section, stating something like "To guarantee code quality of the master branch, all but minor changes should go through pull requests reviewed by peer committers."